README.md

```diff
@@ -16,12 +16,26 @@ require branches to be up to date, require status checks to pass).
 
 ## Usage
 
-The Action supports two run triggers:
+The Action supports three run triggers:
```

Comment:
[four](https://github.com/ridedott/merge-me-action/blob/master/src/index.ts#L32)

Reply:
But how?... I mean, even `GITHUB_TOKEN` is read-only in `pull_request` trigger, right?